wpilibj/src/main/java/edu/wpi/first/wpilibj/shuffleboard/ShuffleboardContainer.java

```diff
@@ -62,7 +62,7 @@ default ShuffleboardLayout getLayout(String title, LayoutType layoutType) {
    * @return the layout with the given title
    * @throws NoSuchElementException if no layout has yet been defined with the given title
    */
-  ShuffleboardLayout getLayout(String title) throws NoSuchElementException;
```

Comment:
Should the @throws docs also be removed?

Reply:
https://pmd.github.io/latest/pmd_rules_java_design.html#avoiduncheckedexceptionsinsignatures recommends removing the throws and adding @throws docs.